### DISCUSSION TITLE
Is there a way to link the local filesystem?

### DISCUSSION THREAD

**Discussion by iwishiwasaneagle** [Cinnamon/kotaemon#338] [Q&A]
I would like to use kotaemon on my notes database that I have collected over 3 years of my PhD in markdown format. However, this is a live dataset and re-uploading changes every day to kotaemon is not feasible. Is there a way to do, for example, a daily upload for a directory and all it's files?

**Comment by boredPanda96** [comment]
+1, I would love to see this feature. I add a bunch of local directories and kotaemon would see change to any files, re index them automatically and all i do is chat with my files

**Reply by chaletsamont** [reply]
Agree!

**Reply by taprosoft** [reply]
It is a work-in-progress feature but rest assured that this is in our back log :D

**Reply by r0kk** [reply]
Do you have any timeline for this? It would be cool to see roadmap you prepared if it is not too private.
